subworkflows/local/utils_nfcore_mag_pipeline/main.nf

```diff
@@ -118,6 +118,11 @@ workflow PIPELINE_INITIALISATION {
     //
     validateInputParameters(
         hybrid
```

Comment:
Uhm I think parentheses and brackets were mixed/killed here somewhere and it won't let me run my test

Reply:
Not sure if here or somewhere else in this file...